scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -267,8 +267,8 @@ trait Config {
    */
   def addReducerEstimator(clsName: String): Config =
     update(Config.ReducerEstimators) {
-      case None => Some(clsName) -> ()
-      case Some(lst) => Some(clsName + "," + lst) -> ()
+      case None => Some(clsName) -> Unit
```

Comment:
@johnynek I think I'm mixing two PR. One is to remove as many warnings as possible(https://github.com/twitter/scalding/issues/1139). For that purpose I'm seeing scala 2.11 gives warning when using () instead of Unit

[warn] /Users/exie/learn/ScaldingGithub/scalding/scalding-core/src/main/scala/com/twitter/scalding/Config.scala:230: Adaptation of argument list by inserting () has been deprecated: this is unlikely to be what you want.
[warn]         signature: ArrowAssoc.->[B](y: B): (A, B)
[warn]   given arguments: <none>
[warn]  after adaptation: ArrowAssoc.->((): Unit)
[warn]       case Some(lst) => Some(clsName + "," + lst) -> ()


I googled around and found this:
http://stackoverflow.com/questions/25351988/adaptation-of-argument-list-by-inserting-has-been-deprecated